plugins/lsp/language-servers/ltex-settings.nix

```diff
@@ -212,52 +215,50 @@ with lib;
-        This setting is language-specific, so use an attrs of the format
-        ```nix
-          {
-            "<LANGUAGE1>" = [
-              "<JSON1>"
-              "<JSON2>"
-              ...
-            ];
-            "<LANGUAGE2>" = [
-              "<JSON1>"
-              "<JSON2>"
-            ];
-            ...
-          };
-        ```
-        where `<LANGUAGE>` denotes the language code in `settings.language` and `<JSON>` is a JSON
-        string containing information about the rule and sentence.
-
-        Although it is possible to manually edit this setting, the intended way is the
-        `Hide false positive` quick fix.
-
-        The JSON string currently has the form `{"rule": "<RULE>", "sentence": "<SENTENCE>"}`, where
-        `<RULE>` is the ID of the LanguageTool rule and `<SENTENCE>` is a Java-compatible regular
-        expression.
-        All occurrences of the given rule are hidden in sentences (as determined by the LanguageTool
-        tokenizer) that match the regular expression.
-        See the documentation for details.
-
-        This setting is a multi-scope setting. See the documentation for details.
-        This setting supports external files. See the documentation for details.
-        If this list is very large, performance may suffer.
-
-        Example:
-        ```nix
-          {
-            "en-US" = [ ":/path/to/externalFile.txt" ];
-          }
-        ```
-      '';
+  hiddenFalsePositives = defaultNullOpts.mkAttrsOf (with types; listOf (maybeRaw str)) { } ''
+    Lists of false-positive diagnostics to hide (by hiding all diagnostics of a specific rule
+    within a specific sentence).
+    This setting is language-specific, so use an attrs of the format
+    ```nix
+      {
+        "<LANGUAGE1>" = [
+          "<JSON1>"
+          "<JSON2>"
+          ...
+        ];
+        "<LANGUAGE2>" = [
+          "<JSON1>"
+          "<JSON2>"
+        ];
+        ...
+      };
+    ```
+    where `<LANGUAGE>` denotes the language code in `settings.language` and `<JSON>` is a JSON
+    string containing information about the rule and sentence.
+
+    Although it is possible to manually edit this setting, the intended way is the
+    `Hide false positive` quick fix.
+
+    The JSON string currently has the form `{"rule": "<RULE>", "sentence": "<SENTENCE>"}`, where
+    `<RULE>` is the ID of the LanguageTool rule and `<SENTENCE>` is a Java-compatible regular
+    expression.
+    All occurrences of the given rule are hidden in sentences (as determined by the LanguageTool
+    tokenizer) that match the regular expression.
+    See the documentation for details.
+
+    This setting is a multi-scope setting. See the documentation for details.
+    This setting supports external files. See the documentation for details.
+    If this list is very large, performance may suffer.
+
+    Example:
+    ```nix
+      {
+        "en-US" = [ ":/path/to/externalFile.txt" ];
+      }
+    ```
```

Comment:
Can we promote this to be a "real" example?

Reply:
Good idea ! Done.